app/routes.py

```diff
@@ -1,2 +1,63 @@
-from flask import Blueprint
+from app import db
+from app.models.planet import Planet
+from flask import Blueprint, jsonify, request, make_response
+
+planets_bp = Blueprint('planets', __name__, url_prefix="/planets")
+
+@planets_bp.route("", methods=["POST", "GET"])
+def handle_all_planets():
+  if request.method == "POST":
+    request_body = request.get_json()
+    new_planet = Planet(name=request_body["name"], description = request_body["description"], circum = request_body["circum"])
+    
+    db.session.add(new_planet)
+    db.session.commit()
+    return make_response(f"Planet {new_planet.name} create", 201)
```

Comment:
Consider returning information about the planet created (its dictionary structure) rather than just a message here. The caller would probably like to at least know the id of the created record. Returning a dictionary with the id and message as two separate parts can make it easy to get the id programmatically, as well as a potentially human friendly message.

Also, even for plain messages, be sure to jsonify them so that the type of the response body is set properly. `make_response` will mark the response body as text/html, when we want it to be application/json.